libsolidity/interface/CompilerStack.h

```diff
@@ -424,6 +432,8 @@ class CompilerStack: public langutil::CharStreamProvider, public evmasm::Abstrac
 		util::LazyInit<Json const> transientStorageLayout;
 		util::LazyInit<Json const> userDocumentation;
 		util::LazyInit<Json const> devDocumentation;
+		util::LazyInit<Json const> ethdebug;
+		util::LazyInit<Json const> ethdebugRuntime;
```

Comment:
This might be problematic (see https://github.com/ethereum/solidity/pull/15451#issuecomment-2371706002) @cameel  just noted that in https://github.com/ethereum/solidity/pull/15368#discussion_r1781701117 - so I guess it would really make sense to change that as sooooon as possible